builtin/settingtypes.txt

```diff
@@ -594,6 +602,19 @@ enable_auto_exposure (Enable Automatic Exposure) bool false
 #    Requires: enable_post_processing, enable_auto_exposure
 exposure_compensation (Exposure compensation) float 0.0 -1.0 1.0
 
+#   Set the post processing gamma value.
+#   Higher values give lower contrast and vice versa.
+#   Range: from 1.0 to 5.0
+#   Default: 1.6
+#
+#   Requires: enable_post_processing, tone_mapping
+secondstage_gamma (Gamma) float 1.6 1.0 5.0
```

Comment:
how does this differ from `display_gamma`?

Reply:
`display_gamma` affects lighting calculations (as in dark caves or light sources) whereas this pertains to post processing. They do entirely different things, so I would say we need both.

Reply:
Sure, but I was thinking both solve the "everything is too dark" use case, so `display_gamma` could eventually go.
Not a topic for this PR anyway.